src/gui/scripts/prepare.js

```diff
@@ -43,6 +43,9 @@ const js = async () => {
     define: {
       'process.env.NODE_ENV': `"${argv.production ? 'production' : 'development'}"`,
     },
+    loader: {
+      '.png': 'file',
+    },
```

Comment:
We don't need to add this unless we use `import` for the images which I didn't see in the code.

Everything in `public/` is copied to `dist/`, so if we only reference them as strings in `src` attributes as is done in this PR, we can remove the loader.